.github/workflows/publish-playground.yml

```diff
@@ -31,23 +31,18 @@ jobs:
         run: rustup target add wasm32-unknown-unknown
       - uses: actions/setup-node@cdca7365b2dadb8aad0a33bc7601856ffabcc48e # v4
         with:
-          node-version: 20
+          node-version: 22
           cache: "npm"
           cache-dependency-path: playground/package-lock.json
-      - uses: jetli/wasm-pack-action@0d096b08b4e5a7de8c28de67e11e945404e9eefa # v0.4.0
-        with:
-          version: v0.13.1
```

Comment:
We now install wasm-pack with NPM. This also removes the need to version it manually in the workflow